packages/@lwc/engine-core/src/framework/template.ts

```diff
@@ -64,6 +48,18 @@ export interface Template {
     };
 }
 
+export let isUpdatingTemplate: boolean = false;
+
+let vmBeingRendered: VM | null = null;
+export function getVMBeingRendered(): VM | null {
+    return vmBeingRendered;
+}
+export function setVMBeingRendered(vm: VM | null) {
+    vmBeingRendered = vm;
+}
+
+export { registerTemplate };
```

Comment:
This export isn't being used anywhere currently, at least it is not imported from template.ts